packages/lb-components/src/components/NLPToolView/textContent/index.tsx

```diff
@@ -77,9 +78,12 @@ const TextContent: React.FC<IProps> = (props) => {
     onSelectionChange,
     textAnnotation,
     remark,
+    activeToolPanel,
   } = props;
   const { enableRemark, displayRemarkList } = remark || {};
 
+  const hideRemark = activeToolPanel ? activeToolPanel !== TOOL_PANEL_KEY.Remark : false;
```

Comment:
You should use event emitter instead of adding params to control the inner state

Reply:
Temp implement, improve it later